model/docs/example_with_datasets.qmd

```diff
@@ -260,6 +259,59 @@ out = hosp_model.plot_posterior(
 )
 ```
 
+Test `posterior_predictive` and `prior_predictive` methods to generate posterior and prior predictive samples.
+
+```{python}
+# | label: demonstrate-use-of-predictive-methods
+import arviz as az
+
+idata = az.from_numpyro(
+    hosp_model.mcmc,
+    posterior_predictive=hosp_model.posterior_predictive(
+        n_timepoints_to_simulate=len(dat["daily_hosp_admits"])
+    ),
+    prior=hosp_model.prior_predictive(
+        n_timepoints_to_simulate=len(dat["daily_hosp_admits"]),
+        numpyro_predictive_args={"num_samples": 1000},
+    ),
+)
+```
+We will use `plot_lm` method from ArviZ to plot the posterior predictive distribution and observed data below:
+```{python}
+# | label: fig-posterior-predictive
+# | fig-cap: Hospital Admissions posterior distribution with plot_lm
+fig, ax = plt.subplots()
+az.plot_lm(
+    "negbinom_rv",
+    idata=idata,
+    kind_pp="hdi",
+    y_kwargs={"color": "black"},
+    y_hat_fill_kwargs={"color": "C0"},
+    axes=ax,
+)
+
+ax.set_title("Posterior Predictive Plot")
+ax.set_ylabel("Hospital Admissions")
+ax.set_xlabel("Days")
+plt.show()
+```
+
+We can assess model fit by comparing the model prediction with the observed data. Below we use the [`plot_ppc`](https://python.arviz.org/en/latest/api/generated/arviz.plot_ppc.html) to compare the posterior predictive samples with the observed data for the first two time steps. This allows us to check how well the model predictions approximate observed data.
```

Comment:
Sphinx doesn't like putting links on `code` text for some reason.
![CleanShot 2024-07-08 at 13 43 09@2x](https://github.com/CDCgov/multisignal-epi-inference/assets/8095080/e5ae981e-1551-426b-8b9a-67605a056f30)

Reply:
Oops Sorry! I think I undid the change to this from my merge